modules/services/karabiner-dk/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.karabiner-dk;
+
+  parentAppDir = "/Applications/Nix Apps";
+in
+{
+  meta.maintainers = [ lib.maintainers.auscyber or "auscyber" ];
+  options.services.karabiner-dk = {
+    enable = mkEnableOption "Karabiner-DK";
+    package = mkPackageOption pkgs "karabiner-dk" { };
+  };
+  config = mkIf cfg.enable {
+    environment.systemPackages = [
+      cfg.package
+    ];
+    environment.extraAppPaths = [
+      "/Library/Application Support/org.pqrs/Karabiner-DriverKit-VirtualHIDDevice"
+    ];
```

Comment:
What is the use case when a person which uses this nix module (which already manages the daemon automatically) would run `/Library/Application Support/org.pqrs/Karabiner-DriverKit-VirtualHIDDevice/Applications/Karabiner-VirtualHIDDevice-Daemon.app/Contents/MacOS/Karabiner-VirtualHIDDevice-Daemon` manually? We don't need to change neither `system.build.applications` nor `system.activationScripts.applications` if there isn't any